.github/workflows/ci-e2e-cassandra.yml

```diff
@@ -45,6 +49,8 @@ jobs:
     - name: Run cassandra integration tests
       id: test-execution
       run: bash scripts/cassandra-integration-test.sh ${{ matrix.version.major }} ${{ matrix.version.schema }} ${{ matrix.jaeger-version }}
```

Comment:
This is creating [duplicate run names](https://github.com/jaegertracing/jaeger/actions/runs/12072885572/job/33667831589?pr=5922) 

We need to include schema dimension. But I don't like true/false in the name. Can you instead have a `schema` matrix variable with values `manual` and `auto`, and then translate those into true/false env var as needed?

Reply:
Done.